model_clone/tests/test_clone_mixin.py

```diff
@@ -649,3 +650,10 @@ def test_cloning_multiple_instances_with_autocommit_is_valid(self):
                 clone.first_name,
                 r"{}\s[\d]".format(Author.UNIQUE_DUPLICATE_SUFFIX),
             )
+
+    def test_cloning_model_with_unique_text_field(self):
+        product = Product.objects.create(name="Test Product")
+        clone = product.make_clone()
+        self.assertEqual(
+            clone.name, f"{product.name} {Product.UNIQUE_DUPLICATE_SUFFIX} 1"
```

Comment:
Seems this line is failing on python2.7 which doesn't support `f-strings`

![Screen Shot 2021-05-17 at 1 41 51 PM](https://user-images.githubusercontent.com/17484350/118533211-cdcea400-b715-11eb-9129-667ae537a522.png)

Reply:
Apologies, I didn't know you still supported Python 2 (and neither did I know that those were a Python 3 feature).

Reply:
No worries. Thanks